include/deal.II/base/symmetric_tensor.h

```diff
@@ -1217,14 +1217,14 @@ namespace internal
       case 2:
         return (data[0] * sdata[0] +
                 data[1] * sdata[1] +
-                Number(2.) * data[2] * sdata[2]);
+                2. * data[2] * sdata[2]);
       default:
         // Start with the non-diagonal part to avoid some multiplications by
         // 2.
         Number sum = data[dim] * sdata[dim];
         for (unsigned int d=dim+1; d<(dim*(dim+1)/2); ++d)
           sum += data[d] * sdata[d];
-        sum *= 2.;
```

Comment:
@davydden I agree on this suggestion. The only slight challenge is that we need to introduce it for all 7 types of `VectorizedArray` (default, SSE float + double, AVX float + double, AVX-512 float + double). ;-)